package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "@google-cloud/common": "^0.12.0",
     "continuation-local-storage": "^3.1.4",
+    "extend": "^3.0.0",
```

Comment:
Why not use https://github.com/sindresorhus/object-assign, it will use the native Object.assign when present (afaik you only need the polyfill for 0.12)